engine/runtime-integration-tests/src/test/java/org/enso/interpreter/test/asserts/AssertionsTest.java

```diff
@@ -49,6 +49,17 @@ public void resetOutput() {
     out.reset();
   }
 
+  @Test
+  public void jvmAssertionsAreEnabled() {
```

Comment:
I have spent ridiculous amount of time debugging `RuntimeErrorsTest` just to realize that I have not enabled JVM assertions, so the DataFlow errors did not contain stack traces. Let's add this test so that nobody encounters something similar in the future.